### PR TITLE
Regenerate release artifacts (2026-07-22): w3id IRIs, metadata, EL-valid

### PR DESCRIPTION
**Pull request by turbomam**
Regenerates the published release artifacts with the corrected build config from #172. This is the first build that produces w3id IRIs + full metadata + an EL-valid, slim ontology, replacing the stale committed artifacts.

## Verified in the built `aio.owl`
- Ontology IRI `https://w3id.org/aio/aio.owl`, versionIRI `https://w3id.org/aio/releases/2026-07-22/aio.owl`
- `aio-full.owl` / `aio-base.owl` IRIs match the w3id `.htaccess` products
- OWL 2 EL profile: in profile (owl, full, base)
- Ontology header: description, 4 creators, citation, created, source, seeAlso, 8 `IAO:0000700` roots
- No import closure (OBI/RO/APOLLO gone); 442 classes; `aio-src.csv` still agrees with the source sheet

## After merging
Create a GitHub release tagged **`v2026-07-22`** (matching the versionIRI date), per `README-editors.md`.

Built on `odkfull:v1.6.1` via `make components-from-new-input prepare_release`.

## Diff vs last release
Axiom-level comparison against the last published release (v2024-11-08): [`docs/release-diffs/aio-diff_2024-11-08_to_2026-07-22.md`](docs/release-diffs/aio-diff_2024-11-08_to_2026-07-22.md).

### CODE REVIEW

File: src/ontology/aio-src.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ID,Category,Synomyms,Related synonyms,Name,ExactSynonym and label identical,description jaccard diff,Legacy Description,description details comment,Claude opus Aristotelian definition,Curated Claude opus Aristotelian definition,Reference (DOI),Layers comment,layer parts,Parent
+AIO:OutputLayer,Category,Synomyms,Related synonyms,Name,ExactSynonym and label identical,description jaccard diff,Legacy Description,description details comment,Claude opus Aristotelian definition,Curated Claude opus Aristotelian definition,Reference (DOI),Layers comment,layer parts,Parent
```
[the remark this answers is not in the document]
Comment (turbomam) [reply src/ontology/aio-src.csv]: Fixed in the clean-room regeneration. The CSV now has the correct two-line template head: line 1 human header (`ID,Category,Synomyms,...`), line 2 ROBOT directives (`ID,AI oio:inSubset  SPLIT=|,...`). No data row on line 1.

File: aio-src.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ID,Category,Synomyms,Related synonyms,Name,ExactSynonym and label identical,description jaccard diff,Legacy Description,description details comment,Claude opus Aristotelian definition,Curated Claude opus Aristotelian definition,Reference (DOI),Layers comment,layer parts,Parent
+AIO:OutputLayer,Category,Synomyms,Related synonyms,Name,ExactSynonym and label identical,description jaccard diff,Legacy Description,description details comment,Claude opus Aristotelian definition,Curated Claude opus Aristotelian definition,Reference (DOI),Layers comment,layer parts,Parent
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio-src.csv]: Fixed in the clean-room regeneration. The CSV now has the correct two-line template head: line 1 human header (`ID,Category,Synomyms,...`), line 2 ROBOT directives (`ID,AI oio:inSubset  SPLIT=|,...`). No data row on line 1.

File: src/ontology/aio-src.csv [hunk 3]
```diff
@@ -107,13 +107,13 @@ AIO:InputLayer,AIO:LayerSubset,,,Input Layer,FALSE,0.52,"The input layer of a ne
...
+AIO:StackedRNNCellsLayer,AIO:LayerSubset,,,StackedRNNCells Layer,,,Memory Cell Layer,FALSE,#N/A,A layer that allows a stack of RNN cells to behave as a single cell.,https://www.tensorflow.org/api_docs/python/tf/keras/layers/StackedRNNCells,,,AIO:Layer
+AIO:ActivationLayer,AIO:LayerSubset,,,Activation Layer,,,Weighted Layer,FALSE,#N/A,A layer that applies an activation function to an output.,https://www.tensorflow.org/api_docs/python/tf/keras/layers/Activation,,,AIO:Layer
+AIO:RegularizationLayer,AIO:LayerSubset,,,Regularization Layer,,,AbstractRNNCell,FALSE,0.36,A layer that applies penalties on layer parameters or layer activity during optimization summed into the loss function that the network optimizes.,https://keras.io/api/layers/regularizers/,,,AIO:Layer
```
[the remark this answers is not in the document]
Comment (turbomam) [reply src/ontology/aio-src.csv]: Fixed. The shift came from formula-driven values leaking into data columns in the source Google Sheet. The sheet was de-formula'd and rebuilt from scratch; the regenerated CSV has no shifted rows and the built `aio.owl` has zero `rdfs:comment "FALSE"`.

File: aio-src.csv [hunk 4]
```diff
@@ -107,13 +107,13 @@ AIO:InputLayer,AIO:LayerSubset,,,Input Layer,FALSE,0.52,"The input layer of a ne
...
+AIO:StackedRNNCellsLayer,AIO:LayerSubset,,,StackedRNNCells Layer,,,Memory Cell Layer,FALSE,#N/A,A layer that allows a stack of RNN cells to behave as a single cell.,https://www.tensorflow.org/api_docs/python/tf/keras/layers/StackedRNNCells,,,AIO:Layer
+AIO:ActivationLayer,AIO:LayerSubset,,,Activation Layer,,,Weighted Layer,FALSE,#N/A,A layer that applies an activation function to an output.,https://www.tensorflow.org/api_docs/python/tf/keras/layers/Activation,,,AIO:Layer
+AIO:RegularizationLayer,AIO:LayerSubset,,,Regularization Layer,,,AbstractRNNCell,FALSE,0.36,A layer that applies penalties on layer parameters or layer activity during optimization summed into the loss function that the network optimizes.,https://keras.io/api/layers/regularizers/,,,AIO:Layer
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio-src.csv]: Fixed. The shift came from formula-driven values leaking into data columns in the source Google Sheet. The sheet was de-formula'd and rebuilt from scratch; the regenerated CSV has no shifted rows and the built `aio.owl` has zero `rdfs:comment "FALSE"`.

File: aio.owl [hunk 5]
```diff
@@ -254,7 +302,7 @@
         <rdfs:subClassOf rdf:resource="https://w3id.org/aio/Layer"/>
         <obo:IAO_0000115>A layer that applies an activation function to an output.</obo:IAO_0000115>
         <oboInOwl:inSubset rdf:resource="https://w3id.org/aio/LayerSubset"/>
-        <rdfs:comment>Applies an activation function to an output.</rdfs:comment>
+        <rdfs:comment>FALSE</rdfs:comment>
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio.owl]: Fixed. `rdfs:comment "FALSE"` no longer appears anywhere in `aio.owl` (0 occurrences) after rebuilding from the de-formula'd sheet.

File: aio.owl [hunk 6]
```diff
@@ -6526,7 +6574,7 @@
         <rdfs:subClassOf rdf:resource="https://w3id.org/aio/Layer"/>
         <obo:IAO_0000115>A layer that applies penalties on layer parameters or layer activity during optimization summed into the loss function that the network optimizes.</obo:IAO_0000115>
         <oboInOwl:inSubset rdf:resource="https://w3id.org/aio/LayerSubset"/>
-        <rdfs:comment>Regularizers allow you to apply penalties on layer parameters or layer activity during optimization. These penalties are summed into the loss function that the network optimizes. Regularization penalties are applied on a per-layer basis.</rdfs:comment>
+        <rdfs:comment>FALSE</rdfs:comment>
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio.owl]: Fixed. `rdfs:comment "FALSE"` no longer appears anywhere in `aio.owl` (0 occurrences) after rebuilding from the de-formula'd sheet.

File: aio.owl [hunk 7]
```diff
@@ -7257,7 +7305,7 @@
         <rdfs:subClassOf rdf:resource="https://w3id.org/aio/Layer"/>
         <obo:IAO_0000115>A layer that allows a stack of RNN cells to behave as a single cell.</obo:IAO_0000115>
         <oboInOwl:inSubset rdf:resource="https://w3id.org/aio/LayerSubset"/>
-        <rdfs:comment>Wrapper allowing a stack of RNN cells to behave as a single cell. Used to implement efficient stacked RNNs.</rdfs:comment>
+        <rdfs:comment>FALSE</rdfs:comment>
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio.owl]: Fixed. `rdfs:comment "FALSE"` no longer appears anywhere in `aio.owl` (0 occurrences) after rebuilding from the de-formula'd sheet.

File: aio.obo [hunk 8]
```diff
@@ -1,23 +1,39 @@
...
+subsetdef: aio:ActivationFunctionSubset ""
+subsetdef: aio:BiasSubset ""
+subsetdef: aio:ClassSubset ""
+subsetdef: aio:FunctionSubset ""
+subsetdef: aio:InstanceNormalizationLayerSubset ""
+subsetdef: aio:LayerSubset ""
+subsetdef: aio:MachineLearningSubset ""
+subsetdef: aio:ModelSubset ""
+subsetdef: aio:NetworkSubset ""
+subsetdef: aio:PreprocessingSubset ""
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio.obo]: Fixed. `subsetdef` lines now carry human-readable names in all three OBO files, e.g. `subsetdef: aio:ActivationFunctionSubset "Activation Function Subset"`.

File: aio-full.obo [hunk 9]
```diff
@@ -1,23 +1,39 @@
...
+subsetdef: aio:ActivationFunctionSubset ""
+subsetdef: aio:BiasSubset ""
+subsetdef: aio:ClassSubset ""
+subsetdef: aio:FunctionSubset ""
+subsetdef: aio:InstanceNormalizationLayerSubset ""
+subsetdef: aio:LayerSubset ""
+subsetdef: aio:MachineLearningSubset ""
+subsetdef: aio:ModelSubset ""
+subsetdef: aio:NetworkSubset ""
+subsetdef: aio:PreprocessingSubset ""
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio-full.obo]: Fixed. `subsetdef` lines now carry human-readable names in all three OBO files, e.g. `subsetdef: aio:ActivationFunctionSubset "Activation Function Subset"`.

File: aio-base.obo [hunk 10]
```diff
@@ -1,25 +1,41 @@
...
+subsetdef: aio:ActivationFunctionSubset ""
+subsetdef: aio:BiasSubset ""
+subsetdef: aio:ClassSubset ""
+subsetdef: aio:FunctionSubset ""
+subsetdef: aio:InstanceNormalizationLayerSubset ""
+subsetdef: aio:LayerSubset ""
+subsetdef: aio:MachineLearningSubset ""
+subsetdef: aio:ModelSubset ""
+subsetdef: aio:NetworkSubset ""
+subsetdef: aio:PreprocessingSubset ""
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio-base.obo]: Fixed. `subsetdef` lines now carry human-readable names in all three OBO files, e.g. `subsetdef: aio:ActivationFunctionSubset "Activation Function Subset"`.

File: src/ontology/aio-edit.owl [hunk 11]
```diff
@@ -86,51 +86,61 @@ AnnotationAssertion(rdfs:label <http://www.geneontology.org/formats/oboInOwl#Sub
...
 
 AnnotationAssertion(rdfs:label aio:NetworkSubset "Network Subset"@en)
+AnnotationAssertion(rdfs:comment aio:NetworkSubset "Network Subset"@en)
 SubAnnotationPropertyOf(aio:NetworkSubset <http://www.geneontology.org/formats/oboInOwl#SubsetProperty>)
 
 # Annotation Property: aio:PreprocessingSubset (aio:PreprocessingSubset)
 
 AnnotationAssertion(rdfs:label aio:PreprocessingSubset "Preprocessing Subset"@en)
+AnnotationAssertion(rdfs:comment aio:PreprocessingSubset "Preprocessing Subset"@en)
 SubAnnotationPropertyOf(aio:PreprocessingSubset <http://www.geneontology.org/formats/oboInOwl#SubsetProperty>)
```
[the remark this answers is not in the document]
Comment (turbomam) [reply src/ontology/aio-edit.owl:153]: Fixed. Added `aio:TrainingStrategySubset` to `aio-edit.owl` (declaration, label, comment, `SubAnnotationPropertyOf oboInOwl:SubsetProperty`), matching the other 10 subset properties, and rebuilt so it now appears in the `subsetdef` header of all three .obo files. Seven classes were already assigned to this subset (Distillation among them after this PR's subset fix) but it had never been declared.

Not adding `IAO:0000115` definitions to `IAO:0000700` / `oboInOwl:SubsetProperty`: those are external OBO vocabulary terms AIO imports, so it should not mint its own definitions for them. That report entry is a WARN on external annotation properties, and the build passes with report fail-on set to ERROR.

File: aio-src.csv [hunk 12]
```diff
@@ -32,7 +32,7 @@ AIO:UncertaintyBias,AIO:BiasSubset,,,Uncertainty Bias,FALSE,0.42,"Bias favoring
...
-AIO:SimponsParadoxBias,AIO:BiasSubset,Simpson's Paradox,,Simpon's Paradox Bias,FALSE,#N/A,A statistical phenomenon where the association between two variables changes when controlling for another variable.,,A selection and sampling bias where the association between two variables changes when controlling for another variable.,A selection and sampling bias where the association between two variables changes when controlling for another variable.,https://doi.org/10.6028/NIST.SP.1270,,,AIO:SelectionAndSamplingBias
+AIO:SimponsParadoxBias,AIO:BiasSubset,Simpson's Paradox,,Simpon's Paradox Bias,FALSE,,A statistical phenomenon where the association between two variables changes when controlling for another variable.,,A selection and sampling bias where the association between two variables changes when controlling for another variable.,A selection and sampling bias where the association between two variables changes when controlling for another variable.,https://doi.org/10.6028/NIST.SP.1270,,,AIO:SelectionAndSamplingBias
```
[the remark this answers is not in the document]
Comment (turbomam) [reply aio-src.csv]: Fixed at source. Corrected the label to `Simpson's Paradox Bias` in the Google Sheet build source and rebuilt. The term ID (`AIO:SimponsParadoxBias`) is unchanged, since IDs are stable and not re-minted from labels.
